pkg/cloud/aws/athenaconfiguration.go

```diff
@@ -12,6 +12,7 @@ type AthenaConfiguration struct {
 	Bucket     string     `json:"bucket"`
 	Region     string     `json:"region"`
 	Database   string     `json:"database"`
+	Catalog    string     `json:"catalog""`
```

Comment:
Looks like there's an extra `"` here for `"catalog""`. Does this affect the serialization/de-serialization of the JSON?

Reply:
Nvm, this is fixed in #2170 !